src/swarm.js

```diff
@@ -726,8 +726,10 @@ SHELL_SCREENRC`
         })
 
         builder.stderr.on('data', (data) => {
-          console.log(chalk.yellow(data.toString().trimEnd()))
```

Comment:
I'm curious ... does this have to change because of the changes above? Or, is it just something you've seen giving you issues?